bindings/GtkSource-5/GtkSource.overrides

```diff
@@ -0,0 +1,182 @@
+namespace GtkSource
+set-attr GtkSource/Buffer/iter_backward_to_context_class_toggle/@parameters/iter transfer-ownership none
+set-attr GtkSource/Buffer/iter_backward_to_context_class_toggle/@parameters/iter direction inout
+set-attr GtkSource/Buffer/iter_backward_to_context_class_toggle/@parameters/iter caller-allocates 1
+
+set-attr GtkSource/Buffer/iter_forward_to_context_class_toggle/@parameters/iter transfer-ownership none
+set-attr GtkSource/Buffer/iter_forward_to_context_class_toggle/@parameters/iter direction inout
+set-attr GtkSource/Buffer/iter_forward_to_context_class_toggle/@parameters/iter caller-allocates 1
+
+set-attr GtkSource/Buffer/backward_iter_to_source_mark/@parameters/iter transfer-ownership none
+set-attr GtkSource/Buffer/backward_iter_to_source_mark/@parameters/iter caller-allocates 1
+set-attr GtkSource/Buffer/forward_iter_to_source_mark/@parameters/iter transfer-ownership none
+set-attr GtkSource/Buffer/forward_iter_to_source_mark/@parameters/iter caller-allocates 1
+
+set-attr GtkSource/Indenter/indent/@parameters/iter transfer-ownership none
+set-attr GtkSource/Indenter/indent/@parameters/iter caller-allocates 1
+
+# Generated from GtkSourceView 5.8.0 with xsltproc Nullable.xslt /usr/share/gir-1.0/GtkSource-5.gir
```

Comment:
Are these nullable annotations needed? Given that the 5.0 version of GtkSourceView is recent we can probably omit these.